bin/asgs-lint

```diff
@@ -597,12 +597,12 @@ sub fatal_configcheck_SCENARIOPACKAGESIZE_if_TROPICALCYCLONE_DEFINED {
             my ( $found_ENSTORM, $ENSTORM ) = $self->_extract_from_string( q{ENSTORM}, $CASE_block );    # attempt to find value of ENSTORM
             my ( $found_PERCENT, $PERCENT ) = $self->_extract_from_string( q{PERCENT}, $CASE_block );    # attempt to find value of PERCENT
 
-            # check for valid start patterns: 'nhcC', 'veerLeft', 'veerRight', 'namforecast'
+            # check for valid start patterns: 'nhcC', 'veerLeft', 'veerRight', 'namforecast', 'gfsforecast'
```

Comment:
vMax, rMax, and nhcOfcl are all possibilities as well

Reply:
nice, I will add them! this is exactly why I needed the review. TY

Reply:
@jasonfleming - I updated this in the latest commit